app/models/user.rb

```diff
@@ -13,4 +13,5 @@ def set_default_role
 
   has_one :volunteer
   has_one :owner
+  has_many :repair
```

Comment:
Should not have has_many :repair .  This line should be removed.  The user object does not have a relationship with the repair object.